js/plugins/google-genai/src/common/types.ts

```diff
@@ -119,7 +119,7 @@ export enum DynamicRetrievalMode {
 /**
  * Schema for parameters passed to {@link FunctionDeclaration.parameters}.
  */
-export interface FunctionDeclarationSchema {
+export declare interface FunctionDeclarationSchema {
```

Comment:
just noticed, Genkit naming conventions reserves `*Schema` suffix for Zod schema. Call this `FunctionDeclaration`.

Reply:
Discussed offline. JSON schema is a reasonable exception to the rule.